source/documentation/other-topics/TZ-Cronjobs.md

```diff
@@ -0,0 +1,53 @@
+### Time Zone Cronjobs
+
+#### Overview
+
+There may be a requirement for some Cronjobs to run at specific times of a day in the UK (including during BST), but Kubernetes Cronjob does not support time zone, our cluster uses UTC timezone internally, and we have no plans to change this. To enable service teams to run cronjobs at specific times, we deployed [TZ Cronjobber][hiddeco-cronjobber](cronjob controller for Kubernetes with support for time zones).
+
+#### Usage
+
+Instead of creating a CronJob like you normally would, you create a TZCronJob, which works exactly the same but supports an additional field: `.spec.timezone`. Set this to the time zone you wish to schedule your jobs in and Cronjobber will take care of the rest.
+
+In the below example timezone is set to "Europe/Amsterdam" and scheduled every day at 14:10, so during BST it will trigger at 13:10, which helps for some cron jobs to run at specific times of a day in the UK (including during BST).
```

Comment:
> so during BST it will trigger at 13:10

This is a bit confusing. What timezone is 13:10 referring to? I'd suggest changing this to a UK example (Europe/London), and clarifying that "during BST the job will trigger at XX:XX UTC, but the rest of the year it will trigger at XX:XX UTC).

Reply:
Updated considering the comment